spec/ar2dto/dto_spec.rb

```diff
@@ -49,7 +49,7 @@ def self.name
     end
 
     context "with options" do
-      let(:options) { { except: :first_name } }
+      let(:options) { { except: "first_name" } }
```

Comment:
Just curious, why these changes happened?

Shouldn't the options be passed as symbols?

Reply:
good question, we probably need to support symbols as well, I'll check

Reply:
just checked, this is the `as_json` Rails method, so I think it's fine